src/bin/gpfdist/regress/input/exttab1.source

```diff
@@ -713,11 +759,80 @@ select count(*) from ext_transform_explicit;
 -- integrity check
 select a from ext_transform_free order by a;
 
+--
+-- Wet tests for DEFAULT and NOT NULL
```

Comment:
Why it's wet? If it's TLA for Write External Table, then it's not common, does not appear in docs, and has not introduced somewhere. Also, TLA written in capitals.

Reply:
Didn't know it was TLA, and yes, it's tests for writable tables, already used [here](https://github.com/arenadata/gpdb/blob/3c9b542f8c33d0e65eddbad878c462a927b81df6/src/bin/gpfdist/regress/input/exttab1.source#L506). I can remove that word since it is already obvious these are tests for writable tables.

Reply:
Okay, you can just s/Wet/WET :)